packages/components/modules/payments/web/hooks/useStripeHook.tsx

```diff
@@ -136,6 +142,51 @@ const useStripeHook = () => {
       mutationKey: ['useCreationSubscription'],
```

Comment:
we should have constant for mutation keys as well. It's fine if you want to add them later, but add at least a todo in all the places that need update